### ISSUE TITLE
[BITV] Notifications menu overlap

### ISSUE THREAD

**Issue by PVince81** [nextcloud/notifications#1268]
When opening the search dropdown or people menu, and then opening the notifications (or the other way around), the notifications menu get overlapped.

We need to find a way to make those menus open/close exclusively.
Maybe moving them to the Vue's Popover would help there ?

**Comment by nickvergessen** [comment]
https://github.com/nextcloud/notifications/blob/master/src/NotificationsApp.vue#L218

Used to do exactly that. Was it broken on server side?
Maybe we need a replacement. Because popovers should not automatically close all other popovers

**Comment by nickvergessen** [comment]
Ref button was not transfered when the HeaderMenu component was introduced.
Should be easily fixable if the OC method still works

**Comment by nickvergessen** [comment]
Seems to work again?
